aws/table_aws_ec2_gateway_load_balancer.go

```diff
@@ -0,0 +1,280 @@
+				Hydrate:     getAwsEc2GatewayLoadBalancerAttributes,
+				Transform:   transform.FromField("Attributes"),
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags attached to the load balancer.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsEc2GatewayLoadBalancerTags,
+				Transform:   transform.FromValue(),
+			},
+
+			// Standard columns
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("LoadBalancerArn").Transform(arnToAkas),
+			},
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("LoadBalancerName"),
+			},
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsEc2ApplicationLoadBalancerTags,
+				Transform:   transform.From(getEc2GatewayLoadBalancerTurbotTags),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listEc2GatewayLoadBalancers(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	plugin.Logger(ctx).Trace("listEc2GatewayLoadBalancers", "AWS_REGION", region)
+
+	// Create Session
+	svc, err := ELBv2Service(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// List call
+	err = svc.DescribeLoadBalancersPages(
+		&elbv2.DescribeLoadBalancersInput{},
+		func(page *elbv2.DescribeLoadBalancersOutput, isLast bool) bool {
+			for _, gatewayLoadBalancer := range page.LoadBalancers {
+				// Filtering the response to return only gateway load balancers
+				if strings.ToLower(*gatewayLoadBalancer.Type) == "gateway" {
+					d.StreamListItem(ctx, gatewayLoadBalancer)
+				}
+			}
+			return !isLast
+		},
+	)
+	return nil, err
+}
+
+//// HYDRATE FUNCTIONS
+
+func getEc2GatewayLoadBalancer(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	//gatewayLoadBalancer := h.Item.(*elbv2.LoadBalancer)
+
+	// Create service
+	svc, err := ELBv2Service(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	quals := d.KeyColumnQuals
+	loadBalancerName := quals["name"].GetStringValue()
+
+	params := &elbv2.DescribeLoadBalancersInput{
+		Names: []*string{aws.String(loadBalancerName)},
+	}
+
+	op, err := svc.DescribeLoadBalancers(params)
+	if err != nil {
+		return nil, err
+	}
+
+	if op.LoadBalancers != nil && len(op.LoadBalancers) > 0 && strings.ToLower(*op.LoadBalancers[0].Type) == "gateway" {
+		return op.LoadBalancers[0], nil
+	}
```

Comment:
Do we need to add filter on `Type`?
Can we create multiple loadbalancers with same name in a specific region?

`Gateway Load Balancer`
![image](https://user-images.githubusercontent.com/38218418/109920803-dac82380-7ce0-11eb-9569-8d251306be5d.png)

`Network Load Balancer`
![image](https://user-images.githubusercontent.com/38218418/109920914-fc290f80-7ce0-11eb-9906-a7d0b307920c.png)